stratum/hal/lib/common/p4_service.cc

```diff
@@ -209,6 +209,10 @@ ::grpc::Status ToGrpcStatus(const ::util::Status& status,
 void LogWriteRequest(uint64 node_id, const ::p4::v1::WriteRequest& req,
                      const std::vector<::util::Status>& results,
                      const absl::Time timestamp) {
+  // Skip logging if flag is empty.
```

Comment:
I'll leave it up to you if this falls under: [https://google.github.io/styleguide/cppguide.html#Implementation_Comment_Donts](https://google.github.io/styleguide/cppguide.html#Implementation_Comment_Donts)